hack/ci/link-check.sh

```diff
@@ -3,18 +3,8 @@ set -ev
 
 CONTAINER_RUN_EXTRA_OPTIONS=${CONTAINER_RUN_EXTRA_OPTIONS:=""}
 CONTAINER_ENGINE=${CONTAINER_ENGINE:="docker"}
-volume_name="olm-html"
 
-function cleanup() {
-    exit_status=$?
-    ${CONTAINER_ENGINE} volume rm ${volume_name}
-    exit $exit_status
-}
-trap cleanup EXIT
-
-${CONTAINER_ENGINE} volume create ${volume_name}
-${CONTAINER_ENGINE} run --rm ${CONTAINER_RUN_EXTRA_OPTIONS} -v "$(pwd):/src" -v ${volume_name}:/src/public klakegg/hugo:0.73.0-ext-ubuntu
 # ignore
 # 1: links going back to help.github.com are rate-limited and can make this flaky
 # 2: docsy autogenerated edit links to original markdown source, which will fail if the markdown file is new
-${CONTAINER_ENGINE} run --rm -v ${volume_name}:/target mtlynch/htmlproofer /target --ignore-empty-alt --ignore-status-codes 429 --allow-hash-href --ignore-urls '/help.github.com/,/edit\/master\/.*\.md/'
+${CONTAINER_ENGINE} run --rm -v $(pwd)/public:/target mtlynch/htmlproofer /target --ignore-empty-alt --ignore-status-codes 429 --allow-hash-href --ignore-urls '/help.github.com/,/edit\/master\/.*\.md/,/tree\/master\/.*\.md/,/new\/master\/.*filename=change-me\.md/'
```

Comment:
This implies that we now have to have a local install of hugo to run, right?  (e.g. `brew install hugo`) How are we capturing that new requirement & workflow?  For e.g., I would always be able to run this script locally to build+validate changes before review, but first I'll have to install hugo now and manually run something?

I guess this goes back to my earlier comment about how close to hugo HEAD we need to be, because it's just 0.101.0 < 0.113.0 that isn't exposed in the available containers.

Reply:
Created #301 to track these other updates needed, so that this PR doesn't get any more complex. 😄

Reply:
Okay. Sounds like we can avoid a massive PR like this if we just update the README then? 

> It talks about a git submodule for docsy, but there is no submodule. Theme commited into the repo. And there is some mention of https://github.com/google/docsy-example.git in readme... Like - how is it supposed to help me run it locally? How is it related?

This is the submodule: https://github.com/operator-framework/olm-docs/tree/master/themes/docsy, which you do remove in a commit further up the chain too. 

Removing the theme from being committed in the repo: what benefit do we get other than saving a little space in the git sever? I.e looking for justification for investing building/review cycles for this effort. 

The mention of  https://github.com/google/docsy-example.git is clearly a typo, if you read the statement above, it's trying to show the example of cloning the olm-docs repo, but with `--git-submodules` on. 

> It assumes that you already have hugo installed.
Looks like there's a missing `Getting started` page that should be pointing to https://gohugo.io/installation/macos/ for Mac installations. 

> It says that you must have version 0.45 (extended). However there is no darwin-arm64 binary of Hugo for versions which work with olm-docs. I tried to compile it myself and gave up. Docker image also fails on darwin-arm64.

It says you must have version 0.45 at least. I am going to try installing Hugo on my machine from that link above and go through the steps to see how things go.